### ISSUE TITLE
[Vue warn]: Failed to resolve component: icon-xmark

### ISSUE THREAD

**Issue by nyxb** [kissu/nuxt3-unplugin-icons#1]
hey i hope you can help me, i get this error

[Vue warn]: Failed to resolve component: icon-xmark
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.
[nitro] [dev] [unhandledRejection] TypeError: Invalid value used as weak map key
    at WeakMap.set (<anonymous>)

**Comment by kissu** [comment]
Solved it? 🤗

**Comment by nyxb** [comment]
> Solved it? 🤗

Not really, but if I'm honest, it's been so long that I don't remember anymore 🤣 I've finally tidied up my issue list.

**Comment by kissu** [comment]
Oh yeah, I missed the fact that it's a hella old issue. 💀 🐵
